test/new-objectid.test.js

```diff
@@ -0,0 +1,257 @@
+// Copyright IBM Corp. 2013,2020. All Rights Reserved.
+// Node module: loopback-connector-mongodb
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+require('./init.js');
+
+let Book, Chapter;
+const ds = global.getDataSource();
+const objectIDLikeString = '7cd2ad46ffc580ba45d3cb1f';
+const objectIDLikeString2 = '7cd2ad46ffc580ba45d3cb1e';
+const promisify = require('bluebird').promisify;
+
+
+describe.only('New ObjectID', function() {
+
+  const Book = ds.createModel(
+    'Book',
+    {
+      d: {id: true, type: String, mongodb: {dataType: 'objectID'}},
+      oId: {type: String, mongodb: {dataType: 'objectID'}},
+      oIds: {type: [String], mongodb: {dataType: 'objectID'}},
+      title: String,
+      strId: String,
+      strIds: [String],
+      authorId: {type: [String], mongodb: {dataType: 'objectID'}}
+    }
+  );
+
+  const Author = ds.createModel(
+    'Author',
+    {
+      id: {id: true, type: String, mongodb: {dataType: 'objectID'}},
```

Comment:
nitpick: according to "if specified with `{id: true}`, need not add `mongodb: {dataType: 'ObjectID'}` (it will be automatically inferred)" the mongodb config here can be omitted.